index.js

```diff
@@ -42,17 +49,17 @@ const expressions = {
     expr: /^\/$/,
     handler: () => BASE_SCC_URL,
   },
+  authReg: {
+    expr: /\/iii\/cas\/login/,
+    handler: (match, parsedURL) => parsedURL.href.replace('catalog.nypl.org', 'ilsstaff.nypl.org'),
+  },
   searchRegWith: {
-    expr: /\/search(~S\d*)?\/([a-zA-Z])?((\w|\W)+)/,
+    expr: /\/search(~S\d*)?\/([a-zA-Z])((\w|\W)+)/,
     handler: match => `${BASE_SCC_URL}search?q=${recodeSearchQuery(match[3])}${getIndexMapping(match[2])}`
   },
-  // searchRegNone: {
-  //     expr: /\/search(~S\d*)?\/?([a-zA-Z])?$/,
-  //     handler: match => `${BASE_SCC_URL}search?q=''${getIndexMapping(match[2])}`
-  // },
   searchRegWithout: {
-    expr: /\/search(~S\d*)?\/([a-zA-Z])?/,
-    handler: (match, parsedURL) => `${BASE_SCC_URL}search?q=${getQueryFromParams(match[0], parsedURL)}${getIndexMapping(match[2])}`
+    expr: /\/search(~S\d*)?(\/([a-zA-Z]))?/,
```

Comment:
Again, if you only need the grouping, not the match, you can use a non-matching group. But I'm actually not sure what this is supposed to match. An example in the comments would be great! Also happy to have a chat.